webapp/lib/view.dart

```diff
@@ -427,15 +430,27 @@ class ContentView {
     });
   }
 
-  void set stale (bool staleState) {
-    if (staleState) {
-      _conversationCharts.forEach((chart) => chart.classes.add('stale'));
-    } else {
-      _conversationCharts.forEach((chart) => chart.classes.remove('stale'));
+  void setStale (String type, bool staleState) {
+    switch (type) {
+      case 'needs_reply_metrics':
```

Comment:
I think you have access to these strings by using `controller.NEEDS_REPLY_METRICS_ROOT_COLLECTION_KEY` and `controller.SYSTEM_METRICS_ROOT_COLLECTION_KEY` ?